### DISCUSSION TITLE
Can't install Atom on Windows 7

### DISCUSSION THREAD

**Discussion by AmazonessKR** [atom/atom#22583] [Q&A]
Here's the Squirrel-Install log:
[Squirrel-Install.log](https://github.com/atom/atom/files/6641493/Squirrel-Install.log)

It just crashes as soon as the installer starts. I followed some instructions on the internet with anti-virus, but eh, none of that really helped.
![image](https://user-images.githubusercontent.com/53908162/121761958-f3b65100-cae7-11eb-807a-ca91d6b1dddc.png)

**Comment by UziTech** [comment]
Windows 7 EOL (End of Life) was over a year ago. I don't think That OS will be supported.

**Reply by AmazonessKR** [reply]
Maybe. I just fresh installed W7 to a new PC, but Atom ran no problem and updated in my old W7 PC.

**Comment by AmazonessKR** [comment]
I ended up just using the pack version. The problem is the install .exe, but the pack version works just fine.
